packages/nodes-base/nodes/HttpRequest/HttpRequest.node.ts

```diff
@@ -14,14 +14,15 @@ export class HttpRequest extends VersionedNodeType {
 			group: ['output'],
 			subtitle: '={{$parameter["requestMethod"] + ": " + $parameter["url"]}}',
 			description: 'Makes an HTTP request and returns the response data',
-			defaultVersion: 4,
+			defaultVersion: 5,
 		};
 
 		const nodeVersions: IVersionedNodeType['nodeVersions'] = {
 			1: new HttpRequestV1(baseDescription),
 			2: new HttpRequestV2(baseDescription),
 			3: new HttpRequestV3(baseDescription),
 			4: new HttpRequestV3(baseDescription),
+			5: new HttpRequestV3(baseDescription),
```

Comment:
considering that v4 was added only 3 days ago, and isn't out in any release, maybe we should use v4 for this check as well.